docs/rst/statistics_backend/get_data.rst

```diff
@@ -0,0 +1,118 @@
++-------------------------------+------------------------------------+------------------------------------+
+| |RTPS_PACKETS_LOST-api|       | |PARTICIPANT-api|                  | |LOCATOR-api|                      |
++-------------------------------+------------------------------------+------------------------------------+
+| |RTPS_BYTES_LOST-api|         | |PARTICIPANT-api|                  | |LOCATOR-api|                      |
++-------------------------------+------------------------------------+------------------------------------+
+| |RESENT_DATA-api|             | |DATAWRITER-api|                   | Not applicable                     |
++-------------------------------+------------------------------------+------------------------------------+
+| |HEARTBEAT_COUNT-api|         | |DATAWRITER-api|                   | Not applicable                     |
++-------------------------------+------------------------------------+------------------------------------+
+| |ACKNACK_COUNT-api|           | |DATAREADER-api|                   | Not applicable                     |
++-------------------------------+------------------------------------+------------------------------------+
+| |NACKFRAG_COUNT-api|          | |DATAREADER-api|                   | Not applicable                     |
++-------------------------------+------------------------------------+------------------------------------+
+| |GAP_COUNT-api|               | |DATAWRITER-api|                   | Not applicable                     |
++-------------------------------+------------------------------------+------------------------------------+
+| |DATA_COUNT-api|              | |DATAWRITER-api|                   | Not applicable                     |
++-------------------------------+------------------------------------+------------------------------------+
+| |PDP_PACKETS-api|             | |PARTICIPANT-api|                  | Not applicable                     |
++-------------------------------+------------------------------------+------------------------------------+
+| |EDP_PACKETS-api|             | |PARTICIPANT-api|                  | Not applicable                     |
++-------------------------------+------------------------------------+------------------------------------+
+| |DISCOVERED_ENTITY-api|       | |PARTICIPANT-api|                  | Not applicable                     |
++-------------------------------+------------------------------------+------------------------------------+
+| |SAMPLE_DATAS-api|            | |DATAWRITER-api|                   | Not applicable                     |
++-------------------------------+------------------------------------+------------------------------------+
+
+
+
+.. _statistics_backend_get_data_examples:
+
+Examples
+^^^^^^^^
+
+Following, some example queries are provided to serve a inspiration for applications using
+*Fast DDS Statistics Backend*.
+
+DataWriter's Fast DDS Latency median example
```

Comment:
**For every example**
Add a possible output for each example.

Reply:
I've included a TODO instead, we'll get there